tests/system.rs

```diff
@@ -66,6 +67,27 @@ fn idempotence_tests() {
     assert!(fails == 0, "{} idempotent tests failed", fails);
 }
 
+// Run rustfmt in tidy mode and check we don't modify the file.
+#[test]
+fn tidy_tests() {
```

Comment:
Yeah, I was getting really lazy at this stage. I'll tidy this up, thanks for keeping me honest :-)